packages/vm/test/api/runTx.spec.ts

```diff
@@ -80,6 +80,141 @@ describe('runTx() -> successful API parameter usage', async () => {
+    let tx = getTransaction(
+      vm.common,
+      TransactionType.FeeMarketEIP1559,
+      true,
+      '',
+      true,
+      0,
+      bytecode,
+      undefined,
+      200000,
+      1
+    )
+
+    const caller = tx.getSenderAddress()
+    const acc = createAccount(BigInt(0), BigInt(1000000000))
+    await vm.stateManager.putAccount(caller, acc)
+
+    let block = Block.fromBlockData(
+      {
+        header: {
+          number: 0,
+          gasLimit: 8000000,
+          baseFeePerGas: '0x1',
+        },
+        transactions: [tx],
+      },
+      { common }
+    )
+    let resBlock = await vm.runBlock({
+      block,
+      generate: true,
+      skipNonce: true,
+      skipBlockValidation: true,
+      skipBalance: false,
+    })
+    let res = resBlock.results[0]
+    const contractAddress = res.createdAddress?.toString()
+
+    /*
+    const code = await vm.stateManager.getContractCode(res.createdAddress as Address)
+    console.log('code', code)
+    */
+
+    // checkpoint
+    await vm.evm.journal.checkpoint()
+
+    // call to "incr"
+    tx = getTransaction(
+      vm.common,
+      TransactionType.FeeMarketEIP1559,
+      true,
+      '',
+      false,
+      1,
+      '0x119fbbd4',
+      contractAddress
+    )
+    block = Block.fromBlockData(
+      {
+        header: {
+          number: 1,
+          gasLimit: 8000000,
+          baseFeePerGas: '0x1',
+        },
+        transactions: [tx],
+      },
+      { common }
+    )
+    resBlock = await vm.runBlock({
+      block,
+      generate: true,
+      skipNonce: true,
+      skipBlockValidation: true,
+      skipBalance: false,
+    })
+
+    // revert
+    await vm.evm.journal.revert()
+
+    // call to "number"
+    tx = getTransaction(
+      vm.common,
+      TransactionType.FeeMarketEIP1559,
+      true,
+      '',
+      false,
+      1,
+      '0x8381f58a',
+      contractAddress
+    )
+    block = Block.fromBlockData(
+      {
+        header: {
+          number: 2,
+          gasLimit: 8000000,
+          baseFeePerGas: '0x1',
+        },
+        transactions: [tx],
+      },
+      { common }
+    )
+    resBlock = await vm.runBlock({
+      block,
+      generate: true,
+      skipNonce: true,
+      skipBlockValidation: true,
+      skipBalance: false,
+    })
+    res = resBlock.results[0]
+
+    assert.equal(
+      bytesToHex(res.execResult.returnValue),
+      '0x0000000000000000000000000000000000000000000000000000000000000000'
```

Comment:
Could you state by adding a "should do x..." comment along `assert()` what you actually want to test here?